scripts/reporting/statsGenElastic/genStats.js

```diff
@@ -0,0 +1,167 @@
+/**
+*	Copyright (C) 2020 3D Repo Ltd
+*
+*	This program is free software: you can redistribute it and/or modify
+*	it under the terms of the GNU Affero General Public License as
+*	published by the Free Software Foundation, either version 3 of the
+*	License, or (at your option) any later version.
+*
+*	This program is distributed in the hope that it will be useful,
+*	but WITHOUT ANY WARRANTY; without even the implied warranty of
+*	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*	GNU Affero General Public License for more details.
+*
+*	You should have received a copy of the GNU Affero General Public License
+*	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict'
+const MongoClient = require('mongodb').MongoClient;
+const UserList = require('./userList');
+const DBStats = require('./dbStats');
+const Utils = require('./utils');
+
+try {
+	for (let i=0; i<10; i++) {
+	}
+  
+	if (errors.length > 0) {
+	  throw errors; 
+	}
+  
+  } catch(e) {
+	  for (let i=0; i<e.length; i++) {
+		console.log(e[i]);
+	  }
+  }
+
+var errors = [];
+
+try {
+	if ( Utils.clean(process.env.ELASTIC_CLOUD_AUTH === undefined ) ) { errors.push(new Error('ELASTIC_CLOUD_AUTH is not set')); } 
+	if ( Utils.clean(process.env.ELASTIC_CLOUD_ID === undefined ) ) { errors.push(new Error('ELASTIC_CLOUD_ID is not set')); }
+	if ( errors.length > 0 ) { throw errors }
+} catch (e) {
+    for (let i=0; i<e.length; i++) {
+		console.log(e[i].message);
+	  }
+	process.exit(-1);
+}
+
+try {
+	process.env.ELASTIC_CLOUD_AUTH.split(":")
```

Comment:
the format is servername:base64 encrypted stuff, if a split doesn't error then I need to test the length of the returned array

Reply:
yea split doesn't error as long as it's type string, it'll just return you the an array of 1 if it can't find the split value 
![image](https://user-images.githubusercontent.com/11945337/98938239-8144aa80-24df-11eb-992e-8df167bb4991.png)